yesod-form/Yesod/Form/Functions.hs

```diff
@@ -105,6 +111,58 @@ askFiles = do
     (x, _, _) <- ask
     return $ liftM snd x
 
+-- | Converts a form field into monadic form 'WForm'. This field requires a
+-- value and will return 'FormFailure' if left empty.
+--
+-- Since 1.4.14
```

Comment:
Minor nitpick, but would you be able to convert this to the new style `@since` comments?